src/vnsw/agent/vrouter/ksync/ksync_flow_memory.cc

```diff
@@ -148,7 +162,7 @@ void KSyncFlowMemory::InitFlowMem() {
     }
 #endif
 
-#if 0 //WINDOWSFIX
+#if !defined(_WIN32)
```

Comment:
I suggest you to use here ifndef like in the rest of the file

Reply:
Original file uses "#if defined" version (see lines: 6, 8).